scripts/scil_smooth_surface.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to smooth a surface with a Laplacian blur.
+
+step_size from 0.1 to 10 is recommended
+Smoothing_time = step_size * nb_steps
+    [1, 10] for a small smoothing
+    [10, 100] for a moderate smoothing
+    [100, 1000] for a big smoothing
+"""
+
+import argparse
+import logging
+
+import numpy as np
+from trimeshpy.io import load_mesh_from_file
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_surface',
+                   help='Input surface (FreeSurfer or supported by VTK).')
+
+    p.add_argument('out_surface',
+                   help='Output smoothed surface (formats supported by VTK).')
+
+    p.add_argument('-m', '--vts_mask',
+                   help='Vertices mask, where to apply the flow (.npy).')
+
+    p.add_argument('-n', '--nb_steps', type=int, default=2,
+                   help='Number of steps for laplacian smooth [%(default)s].')
```

Comment:
I think all our other scilpy scripts use [default: %(default)s].

Can you please add this here too.

Reply:
I look in general and there is more [%(default)s] then  (Default: %(default)s)
recent script also use  [%(default)s] 
@arnaudbore  @frheault ?